src/utils/index.ts

```diff
@@ -65,5 +66,11 @@ export function getApexDomain(window: Window): string {
     }
 
     const split = host.split('.');
-    return `${split[split.length - 2]}.${split[split.length - 1]}`;
+    const hostLength = split.length;
+    const apex =  `${split[hostLength - 2]}.${split[hostLength - 1]}`;
+
+    if (SLDs.indexOf(apex) < 0) {
```

Comment:
I think it's more clear if we check for the positive case here (`>= 0`) because being a multi-level suffix is the exception. It also makes it easier to an explanatory comment for why this case is necessary, which I think would be helpful (e.g. `Handle two-level suffixes such as .co.uk`).